pubspec.yaml

```diff
@@ -5,13 +5,13 @@ homepage: https://github.com/wslaimin
 repository: https://github.com/wslaimin/http_proxy
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
-  flutter: ">=1.10.0"
+  sdk: '>=3.1.3  <4.0.0'
+  flutter: 3.13.6
```

Comment:
not sure if this line is required at all since I don't find a flutter tag under environment in the official documentation https://docs.flutter.dev/tools/pubspec